static/css/cf-enhancements.less

```diff
@@ -115,6 +115,42 @@
     }
 }
 
+/* topdoc
+  name: respond-to-retina() (mixin)
```

Comment:
Yeah, it should probably take a value for the ratio/res you want to target, too, like the `respond-to-min` and `respond-to-max` mixins.